include/blmc_robots/real_finger.hpp

```diff
@@ -83,26 +154,52 @@ class RealFingerDriver : public robot_interfaces::RobotDriver<
         return joint_modules_.get_measured_index_angles();
     }
 
+public:
+    Observation get_latest_observation() override
+    {
+        Observation observation;
+        observation.angle = joint_modules_.get_measured_angles();
+        observation.velocity = joint_modules_.get_measured_velocities();
+        observation.torque = joint_modules_.get_measured_torques();
+        return observation;
+    }
+
 protected:
-    RealFingerDriver(const Motors &motors)
-        : RobotDriver(0.003, 0.005),
-          joint_modules_(motors, 0.02 * Vector::Ones(), 9.0 * Vector::Ones(),
+    static Motors create_motors(const MotorBoards &motor_boards)
+    {
```

Comment:
this will not work in general (see comment above)

Reply:
done